src/main/java/WebRTCTestRunner.java

```diff
@@ -0,0 +1,84 @@
+
+import java.net.URL;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+public class WebRTCTestRunner implements Runnable {
+
+	
+	public static final String USERNAME = System.getenv("BROWSERSTACK_USERNAME");
+   public static final String ACCESS_KEY = System.getenv("BROWSERSTACK_ACCESS_KEY");
+
+    public static final String HUB_URL = "https://" + USERNAME + ":" + ACCESS_KEY + "@hub-cloud.browserstack.com/wd/hub";
```

Comment:
refrain from using USERNAME and ACCESS_KEY in the URL. use browserstack capabilities: browserstack.user and browserstack.key instead

Reply:
done